lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,224 @@
+import 'dart:ui';
+
+import 'package:farmsmart_flutter/ui/common/ActionSheet.dart';
+import 'package:farmsmart_flutter/ui/common/ActionSheetListItem.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+class _Strings {
+  static final englishAction = "English";
+  static final swahiliAction = "Swahili";
+  static final confirmAction = "Confirm";
+  static final cancelAction = "Cancel";
+  static final englishIcon = "assets/icons/flag_usa.png";
```

Comment:
maybe have the icons in their own _Icons class.

Reply:
Perfect !